.github/workflows/release.yml

```diff
@@ -166,215 +131,110 @@ jobs:
-            marker: ''
-            type: oss
-            k8s: 1.19.1
-          - os: ubuntu-20.04
-            image: nginx-ingress
-            tag: ${{ needs.binary.outputs.version }}
-            marker: ''
-            type: oss
-            k8s: 1.18.8
-          - os: ubuntu-20.04
-            image: nginx-ingress
-            tag: ${{ needs.binary.outputs.version }}
-            marker: ''
-            type: oss
-            k8s: 1.17.11
-          - os: ubuntu-20.04
-            image: nginx-ingress
-            tag: ${{ needs.binary.outputs.version }}
-            marker: ''
-            type: oss
-            k8s: 1.16.15
-          - os: ubuntu-20.04
-            image: nginx-ingress
-            tag: ${{ needs.binary.outputs.version }}
-            marker: ''
-            type: oss
-            k8s: 1.15.12
-          - os: ubuntu-20.04
-            image: nginx-plus-ingress
-            tag: ${{ needs.binary.outputs.version }}
-            marker: ''
-            type: plus
-            k8s: 1.19.1
-          - os: ubuntu-20.04
-            image: nginx-plus-ingress
-            tag: ${{ needs.binary.outputs.version }}
-            marker: ''
-            type: plus
-            k8s: 1.18.8
-          - os: ubuntu-20.04
-            image: nginx-plus-ingress
-            tag: ${{ needs.binary.outputs.version }}
-            marker: ''
-            type: plus
-            k8s: 1.17.11
-          - os: ubuntu-20.04
-            image: nginx-plus-ingress
-            tag: ${{ needs.binary.outputs.version }}
-            marker: ''
-            type: plus
-            k8s: 1.16.15
-          - os: ubuntu-20.04
-            image: nginx-plus-ingress
-            tag: ${{ needs.binary.outputs.version }}
-            marker: ''
-            type: plus
-            k8s: 1.15.12
-          - os: ubuntu-20.04
-            image: nginx-plus-ingress
-            tag: ${{ needs.binary.outputs.version }}-ap
-            marker: '-m appprotect'
-            type: plus-ap
-            k8s: 1.19.1
-          - os: ubuntu-20.04
-            image: nginx-plus-ingress
-            tag: ${{ needs.binary.outputs.version }}-ap
-            marker: '-m appprotect'
-            type: plus-ap
-            k8s: 1.18.8
-          - os: ubuntu-20.04
-            image: nginx-plus-ingress
-            tag: ${{ needs.binary.outputs.version }}-ap
-            marker: '-m appprotect'
-            type: plus-ap
-            k8s: 1.17.11
-          - os: ubuntu-20.04
-            image: nginx-plus-ingress
-            tag: ${{ needs.binary.outputs.version }}-ap
-            marker: '-m appprotect'
-            type: plus-ap
-            k8s: 1.16.15
-          - os: ubuntu-20.04
-            image: nginx-plus-ingress
-            tag: ${{ needs.binary.outputs.version }}-ap
-            marker: '-m appprotect'
-            type: plus-ap
-            k8s: 1.15.12
+        k8s: [1.19.1, 1.18.8, 1.17.11, 1.16.15, 1.15.12]
```

Comment:
Can we remove 1.15 and include 1.20 similar to nightly https://github.com/nginxinc/kubernetes-ingress/pull/1601/files#diff-0d5658b415099a82c11c03a06ca4ec765b4003a1f4b2f3f1943980a882cf8aa6L278

Reply:
Oh sure, no problem. I missed that, thanks 👍🏼